pkg/apis/config/v1/defaults_test.go

```diff
@@ -0,0 +1,61 @@
+package v1
+
+import (
+	"github.com/google/go-cmp/cmp"
```

Comment:
Are there no changes to `go.mod`? This package is an indirect dependency at the moment. This would make it a direct dependency AFAIU.

Anyway, could you use `github.com/go-test/deep` instead, as this is what the project uses?